File: tests/fixtures/sample.py
# Sample Python module for codeweight integration tests.


def add(a, b):
    if a < 0:
        return b
    return a + b


def classify(x):
    if x > 10:
        return "big"
    elif x > 5:
        return "mid"
    else:
        return "small"

File: tests/fixtures/sample.rs
// Sample Rust module for codeweight integration tests.

pub fn add(a: i32, b: i32) -> i32 {
    if a < 0 {
        return b;
    }
    a + b
}

pub fn classify(x: i32) -> &'static str {
    if x > 10 {
        "big"
    } else if x > 5 {
        "mid"
    } else {
        "small"
    }
}
